lib/jekyll/configuration.rb

```diff
@@ -11,7 +11,8 @@ class Configuration < Hash
       'destination'   => File.join(Dir.pwd, '_site'),
       'plugins'       => '_plugins',
       'layouts'       => '_layouts',
-      'data_source'   =>  '_data',
+      'data_source'   => '_data',
+      'includes_dir'  => '_includes',
```

Comment:
Sure, I could do that. So you want all directory configuration options to have a `_dir` suffix and a backwards-compatibility layer that maps old names to new names?